Dockerfile

```diff
@@ -172,3 +172,25 @@ EXPOSE 4822
 #
 CMD /usr/local/guacamole/sbin/guacd -b 0.0.0.0 -L $GUACD_LOG_LEVEL -f
```

Comment:
It looks like guacd has already been started here so don't we want to remove this line if we are starting it again in `guacd.sh` ?

Reply:
Only the last CMD actually does anything, so this is overwritten by our /guacd.sh CMD. 

I decided to leave it to keep the file as untouched as possible and we're overriding functionality at the end.

Reply:
ah right ok :+1: 
"There can only be one CMD instruction in a Dockerfile. If you list more than one CMD then only the last CMD will take effect."